.github/workflows/rust.yaml

```diff
@@ -158,15 +152,7 @@ jobs:
             ~/.cargo/git
           key: ${{ matrix.os }}-cargo-${{ hashFiles('**/Cargo.toml') }}
 
-      - name: cargo test (Linux with CUDA)
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --features=subspace-farmer/cuda
```

Comment:
what were these lines doing then, if we cannot run the tests? I thought this was ensuring having a machine with CUDA-GPU...

Reply:
Was it just compiling the code under `cuda` flag, but there were no cuda-gpu, so in the code `#[cfg(cuda)]` was not being followed?

Reply:
It was compiling CUDA code, but if it doesn't find any GPU it just uses CPU code path. Remember, API is no longer split into CPU/CUDA, it is just batching where implementation details are hidden. No GPU - not an issue anymore, API is still the same.

Reply:
Yes I remember the API. I just assumed CI will assign a device with CUDA-gpu because we are requesting CUDA. I was too optimistic it seems :D

Resolving this convo now, makes sense to remove these tests.